packages/e2e-tests/tests/console_warp-01.test.ts

```diff
@@ -31,19 +31,11 @@ test(`console_warp-01: should support warping to console messages`, async ({
   await resumeToLine(page, 12);
   await executeAndVerifyTerminalExpression(page, "number", 5);
 
-  // This error message has different text on gecko vs. chromium.
-  const target = await getRecordingTarget(page);
-  const errorText =
-    target == "gecko" ? "window.foo is undefined" : "Cannot set property 'bar' of undefined";
-  await warpToMessage(page, errorText);
+  await warpToMessage(page, "Cannot set properties of undefined");
   await reverseStepOverToLine(page, 7);
 
   await warpToMessage(page, "superclass", 40);
 
-  if (target == "gecko") {
-    // As above, we need an additional reverse step over in gecko.
-    await reverseStepOverToLine(page, 40);
-  }
-
+  await reverseStepOverToLine(page, 40);
```

Comment:
The comment above said we only needed this in Gecko but it seems we now need it in Chromium as well?